--- origsrc/gthumb-2.10.11/libgthumb/jpegutils/transupp.c	2009-02-24 04:18:15.000000000 -0600
+++ src/gthumb-2.10.11/libgthumb/jpegutils/transupp.c	2009-09-09 00:10:37.629845000 -0500
@@ -49,7 +49,7 @@
 #define MAX(a, b)  (((a) > (b)) ? (a) : (b))
 #endif
 
-#ifdef __CYGWIN__
+#if defined(__CYGWIN__) && JPEG_LIB_VERSION < 70
 #define height_in_blocks height_in_data_units
 #define width_in_blocks  width_in_data_units
 #endif
