catalogs: https://bugzilla.gnome.org/show_bug.cgi?id=630311

--- origsrc/gthumb-2.11.92/extensions/catalogs/gth-organize-task.c	2010-09-13 08:32:28.000000000 -0500
+++ src/gthumb-2.11.92/extensions/catalogs/gth-organize-task.c	2010-09-21 21:55:40.368761800 -0500
@@ -243,6 +243,7 @@ add_catalog_for_date (GthOrganizeTask *s
 	}
 
 	if (catalog == NULL) {
+#ifndef __CYGWIN__
 		if (gth_main_extension_is_active ("search")) {
 			GthTest *date_test;
 			GthTest *test_chain;
@@ -264,6 +265,7 @@ add_catalog_for_date (GthOrganizeTask *s
 			g_object_unref (date_test);
 		}
 		else
+#endif
 			catalog = gth_catalog_new ();
 	}
 
@@ -319,6 +321,7 @@ add_catalog_for_tag (GthOrganizeTask *se
 	}
 
 	if (catalog == NULL) {
+#ifndef __CYGWIN__
 		if (gth_main_extension_is_active ("search")) {
 			GthTest *tag_test;
 			GthTest *test_chain;
@@ -339,6 +342,7 @@ add_catalog_for_tag (GthOrganizeTask *se
 			g_object_unref (tag_test);
 		}
 		else
+#endif
 			catalog = gth_catalog_new ();
 	}
 	gth_catalog_set_file (catalog, catalog_file);
--- origsrc/gthumb-2.11.92/extensions/image_print/Makefile.am	2010-07-15 13:45:39.000000000 -0500
+++ src/gthumb-2.11.92/extensions/image_print/Makefile.am	2010-09-21 22:10:44.482957900 -0500
@@ -19,7 +19,7 @@ libimage_print_la_SOURCES = 		\
 
 libimage_print_la_CPPFLAGS = $(GTHUMB_CFLAGS) -I$(top_srcdir) -I$(top_builddir)/gthumb 
 libimage_print_la_LDFLAGS = $(EXTENSION_LIBTOOL_FLAGS)
-libimage_print_la_LIBADD = $(GTHUMB_LIBS) 
+libimage_print_la_LIBADD = $(GTHUMB_LIBS) ../image_viewer/libimage_viewer.la
 libimage_print_la_DEPENDENCIES = $(top_builddir)/gthumb/gthumb$(EXEEXT)
 
 extensioninidir = $(extensiondir)
