--- origsrc/gthumb-2.11.4/extensions/Makefile.am	2010-06-20 07:49:46.000000000 -0500
+++ src/gthumb-2.11.4/extensions/Makefile.am	2010-06-30 02:24:49.091114700 -0500
@@ -10,16 +10,16 @@ SUBDIRS = 			\
 	exiv2_tools		\
 	export_tools		\
 	facebook		\
+	image_viewer		\
 	file_manager		\
 	file_tools		\
 	file_viewer		\
 	gstreamer_utils		\
 	gstreamer_tools		\
 	image_print		\
-	image_viewer		\
-	importer		\
 	jpeg_utils		\
 	image_rotation		\
+	importer		\
 	list_tools		\
 	oauth			\
 	photo_importer		\
--- origsrc/gthumb-2.11.4/extensions/file_tools/Makefile.am	2010-06-23 12:23:58.000000000 -0500
+++ src/gthumb-2.11.4/extensions/file_tools/Makefile.am	2010-06-30 02:09:16.674783500 -0500
@@ -68,7 +68,7 @@ libfile_tools_la_SOURCES = 		\
 
 libfile_tools_la_CFLAGS = $(GTHUMB_CFLAGS) -I$(top_srcdir) -I$(top_builddir)/gthumb 
 libfile_tools_la_LDFLAGS = $(EXTENSION_LIBTOOL_FLAGS)
-libfile_tools_la_LIBADD = $(GTHUMB_LIBS)
+libfile_tools_la_LIBADD = $(GTHUMB_LIBS) ../image_viewer/libimage_viewer.la
 libfile_tools_la_DEPENDENCIES = $(top_builddir)/gthumb/gthumb$(EXEEXT)
 
 extensioninidir = $(extensiondir)
--- origsrc/gthumb-2.11.4/extensions/flicker/Makefile.am	2010-06-19 15:39:31.000000000 -0500
+++ src/gthumb-2.11.4/extensions/flicker/Makefile.am	2010-06-30 03:23:10.147363700 -0500
@@ -37,7 +37,7 @@ libflicker_la_SOURCES = 			\
 
 libflicker_la_CFLAGS = $(GTHUMB_CFLAGS) $(LIBSOUP_CFLAGS) $(GNOME_KEYRING_CFLAGS) -I$(top_srcdir) -I$(top_builddir)/gthumb 
 libflicker_la_LDFLAGS = $(EXTENSION_LIBTOOL_FLAGS)
-libflicker_la_LIBADD = $(GTHUMB_LIBS) $(LIBSOUP_LIBS) $(GNOME_KEYRING_LIBS)
+libflicker_la_LIBADD = $(GTHUMB_LIBS) $(LIBSOUP_LIBS) $(GNOME_KEYRING_LIBS) ../importer/libimporter.la
 libflicker_la_DEPENDENCIES = $(top_builddir)/gthumb/gthumb$(EXEEXT)
 
 extensioninidir = $(extensiondir)
--- origsrc/gthumb-2.11.4/extensions/importer/Makefile.am	2010-06-19 15:39:31.000000000 -0500
+++ src/gthumb-2.11.4/extensions/importer/Makefile.am	2010-06-30 02:25:56.596975800 -0500
@@ -45,7 +45,7 @@ libimporter_la_SOURCES = 			\
 
 libimporter_la_CFLAGS = $(GTHUMB_CFLAGS) -I$(top_srcdir) -I$(top_builddir)/gthumb 
 libimporter_la_LDFLAGS = $(EXTENSION_LIBTOOL_FLAGS)
-libimporter_la_LIBADD = $(GTHUMB_LIBS)
+libimporter_la_LIBADD = $(GTHUMB_LIBS) ../catalogs/libcatalogs.la ../exiv2_tools/libexiv2_tools.la ../image_rotation/libimage_rotation.la
 libimporter_la_DEPENDENCIES = $(top_builddir)/gthumb/gthumb$(EXEEXT)
 
 extensioninidir = $(extensiondir)
--- origsrc/gthumb-2.11.4/extensions/oauth/Makefile.am	2010-06-19 15:39:31.000000000 -0500
+++ src/gthumb-2.11.4/extensions/oauth/Makefile.am	2010-06-30 02:35:38.057233400 -0500
@@ -20,7 +20,7 @@ liboauth_la_SOURCES = 				\
 
 liboauth_la_CFLAGS = $(GTHUMB_CFLAGS) $(LIBSOUP_CFLAGS) $(GNOME_KEYRING_CFLAGS) -I$(top_srcdir) -I$(top_builddir)/gthumb 
 liboauth_la_LDFLAGS = $(EXTENSION_LIBTOOL_FLAGS)
-liboauth_la_LIBADD = $(GTHUMB_LIBS) $(JPEG_LIBS) $(GNOME_KEYRING_LIBS)
+liboauth_la_LIBADD = $(GTHUMB_LIBS) $(JPEG_LIBS) $(GNOME_KEYRING_LIBS) $(LIBSOUP_LIBS)
 liboauth_la_DEPENDENCIES = $(top_builddir)/gthumb/gthumb$(EXEEXT)
 
 extensioninidir = $(extensiondir)
--- origsrc/gthumb-2.11.4/extensions/picasaweb/Makefile.am	2010-06-19 15:39:31.000000000 -0500
+++ src/gthumb-2.11.4/extensions/picasaweb/Makefile.am	2010-06-30 02:44:34.395910200 -0500
@@ -38,7 +38,7 @@ libpicasaweb_la_SOURCES = 			\
 
 libpicasaweb_la_CFLAGS = $(GTHUMB_CFLAGS) $(LIBSOUP_CFLAGS) $(GNOME_KEYRING_CFLAGS) -I$(top_srcdir) -I$(top_builddir)/gthumb 
 libpicasaweb_la_LDFLAGS = $(EXTENSION_LIBTOOL_FLAGS)
-libpicasaweb_la_LIBADD = $(GTHUMB_LIBS) $(LIBSOUP_LIBS) $(GNOME_KEYRING_LIBS)
+libpicasaweb_la_LIBADD = $(GTHUMB_LIBS) $(LIBSOUP_LIBS) $(GNOME_KEYRING_LIBS) ../importer/libimporter.la
 libpicasaweb_la_DEPENDENCIES = $(top_builddir)/gthumb/gthumb$(EXEEXT)
 
 extensioninidir = $(extensiondir)
--- origsrc/gthumb-2.11.4/extensions/pixbuf_savers/Makefile.am	2010-06-19 15:39:31.000000000 -0500
+++ src/gthumb-2.11.4/extensions/pixbuf_savers/Makefile.am	2010-06-30 02:50:44.296067300 -0500
@@ -47,7 +47,7 @@ libpixbuf_savers_la_SOURCES = 	\
 
 libpixbuf_savers_la_CFLAGS = $(GTHUMB_CFLAGS) $(TIFF_CFLAGS) -I$(top_srcdir) -I$(top_builddir)/gthumb 
 libpixbuf_savers_la_LDFLAGS = $(EXTENSION_LIBTOOL_FLAGS)
-libpixbuf_savers_la_LIBADD = $(GTHUMB_LIBS) $(TIFF_LIBS)
+libpixbuf_savers_la_LIBADD = $(GTHUMB_LIBS) $(TIFF_LIBS) ../jpeg_utils/libjpeg_utils.la
 libpixbuf_savers_la_DEPENDENCIES = $(top_builddir)/gthumb/gthumb$(EXEEXT)
 
 extensioninidir = $(extensiondir)
--- origsrc/gthumb-2.11.4/extensions/webalbums/Makefile.am	2010-06-20 04:00:01.000000000 -0500
+++ src/gthumb-2.11.4/extensions/webalbums/Makefile.am	2010-06-30 03:18:26.954165900 -0500
@@ -34,7 +34,7 @@ endif
 
 libwebalbums_la_CFLAGS = $(GTHUMB_CFLAGS) -I$(top_srcdir) -I$(top_builddir)/gthumb -DWEBALBUM_DATADIR=\"$(webalbum_datadir)\"
 libwebalbums_la_LDFLAGS = $(EXTENSION_LIBTOOL_FLAGS)
-libwebalbums_la_LIBADD = $(GTHUMB_LIBS)
+libwebalbums_la_LIBADD = $(GTHUMB_LIBS) ../image_rotation/libimage_rotation.la
 libwebalbums_la_DEPENDENCIES = $(top_builddir)/gthumb/gthumb$(EXEEXT)
 
 extensioninidir = $(extensiondir)
